src/core/libraries/np_manager/np_manager.cpp

```diff
@@ -2515,12 +2504,26 @@ int PS4_SYSV_ABI Func_FF966E4351E564D6() {
     return ORBIS_OK;
 }
 
+struct NpStateCallbackForNpToolkit {
+    OrbisNpStateCallbackForNpToolkit func;
+    void* userdata;
+};
+
+NpStateCallbackForNpToolkit NpStateCbForNp;
+
+int PS4_SYSV_ABI sceNpCheckCallbackForLib() {
+    // LOG_ERROR(Lib_NpManager, "(STUBBED) called");
+    NpStateCbForNp.func(1, ORBIS_NP_STATE_SIGNED_OUT, NpStateCbForNp.userdata);
```

Comment:
Should use linker `ExecuteGuest` for this.

Reply:
done